FirefoxPrivateNetworkVPNUITests/FirefoxPrivateNetworkVPNUITests.swift

```diff
@@ -79,4 +79,42 @@ class FirefoxPrivateNetworkVPNUITests: BaseTestCase {
         XCTAssertTrue(app.buttons["ReloadButton"].exists)
         XCTAssertTrue(app.webViews.textFields["Email"].exists)
     }
+
+    func testSignInAsNonSubscribedUser() {
+
+        // The main screen is shown
+        waitForExistence(app.staticTexts["Mozilla VPN"], timeout: 5)
+        XCTAssertTrue(app.staticTexts["Mozilla VPN"].exists, "The main page is not loaded correctly")
+
+        // Tap on Get started
+        app.buttons["Get started"].tap()
+
+        // Wait for the FxASingIn page to be shown
+        waitForExistence(app.webViews.textFields["Email"], timeout: 10)
+        app.textFields["Email"].tap()
+        app.typeText("test-f5aefc1935@restmail.net")
+        waitForExistence(app.buttons["Continue"], timeout: 20)
+        app.buttons["Continue"].firstMatch.tap()
+
+        //Enter the password
+        app.secureTextFields["Password"].tap()
+        app.typeText("gkgJqyzJ")
+        app.buttons["Sign in"].tap()
+
+        //Verify that a non subscibed VPN user can see the 'Try Mozilla link' message
+        waitForExistence(app.staticTexts["Mozilla VPN"], timeout: 15)
+        XCTAssertTrue(app.staticTexts["Subscribe to turn on VPN. Try Mozilla VPN"].exists)
+
+       //Sign out
```

Comment:
Here you are going to Settings before signing out. I would modify the comment like // Go to Settings and Sign out, so with one comment is enough and clear.